testgrid/config.yaml

```diff
@@ -1743,6 +1743,29 @@ test_groups:
   gcs_prefix: kubernetes-jenkins/pr-logs/directory/pull-community-verify
   num_columns_recent: 20
 
+# kubernetes-sigs/release-notes
+- name: pull-release-notes-build-dev
+  gcs_prefix: kubernetes-sigs/release-notes/pull-release-notes-build-dev
```

Comment:
This needs to be `kubernetes-jenkins/pr-logs/directory/pull-release-notes-build-dev` (and same for all) :)

Reply:
Oh thank you so much for hinting me into the right direction :pray: